lib/run-tests/worker.js

```diff
@@ -80,6 +80,8 @@ exports.runTest = async function (test) {
   const cacheData = cacheDB?.get(cacheKey);
   if (cacheData?.hash === cacheHash) {
     return cacheData.ret;
+  } else if (cacheData) {
+    console.log(`# Cache hash mismatch for ${file}`);
```

Comment:
This is for debugging https://github.com/babel/babel/pull/17353#issuecomment-2925558721.